src/test/components/PrivateRoute.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { PrivateRoute } from '../../components/PrivateRoute/PrivateRoute';
+import { Dashboard } from '../../views/Dashboard/Dashboard';
+
+const props = {
+  component: Dashboard,
+  isAuthenticated: true,
+};
+describe.only('component: PraveRoute', () => {
```

Comment:
PrivateRoute is not spelled correctly here 
`describe.only('component: PrivateRoute', () => {`

Reply:
Get rid of `describe.only`